frontend/src/components/PlayButton/PlayCard.js

```diff
@@ -1,8 +1,36 @@
+import React, {useEffect, useState} from "react";
 import classNames from "classnames";
 
 import Histogram from "../Histogram/Histogram";
-
+import { MEDIA_ROOT } from "../../config";
 const PlayCard = ({ onClick, registerUserClicks, playing, section }) => {
+    const [firstRender, setFirstRender] = useState(false);
+
+    useEffect(() => {
+        if (!firstRender) {
+          createBuffer();
+          setFirstRender(true);
+        }
+    }, [firstRender])
+    
+    const createBuffer = () => {
+        // preload the section (temporary solution for ICMPC)
+        const audioBuffer = document.createElement("audio");
+        audioBuffer.id = `buffer-${section.id}`;    
+        audioBuffer.src = MEDIA_ROOT + section.url;
+        audioBuffer.crossorigin = "use-credentials";
```

Comment:
I don't think we want to set 'use-credentials' here, since then the external source won't play unless its response headers are set to certain values, which may not always be possible.

Reply:
But will test for myself - of course, external audio will not be a problem anymore if we can preload everything from our own server.